src/Subtitles/SubtitleComponent.js

```diff
@@ -137,11 +153,12 @@ export default class SubtitleComponent extends Lightning.Component {
 
   _defineBreakpoint(text, breakpoint) {
     if (breakpoint >= this.maxWidth) return this.maxWidth
+    const textureDefaults = new Lightning.textures.TextTexture(this.stage).cloneArgs()
```

Comment:
I agree it seems more logical to put this functionality in the method that uses it, but I'd say it's more efficient to keep this line in the `_init()` method so it's only executed once (and not every time the breakpoint is calculated - which happens multiple times)

Reply:
Oh right, forgot to revert that back!